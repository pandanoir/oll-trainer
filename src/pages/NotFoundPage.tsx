import { faSadTear } from '@fortawesome/free-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import 'twin.macro';
import { PrimaryLinkButton } from '../components/common/PrimaryLinkButton';

export const NotFoundPage = () => (
  <div tw="px-3">
    <div tw="text-3xl py-3 border-b-2 mb-3">
      <FontAwesomeIcon icon={faSadTear} /> Not found
    </div>
    <div>すみません、お探しのページが見つかりませんでした…</div>
    <div>
      <PrimaryLinkButton to="/">Home に戻る</PrimaryLinkButton>
    </div>
  </div>
);
