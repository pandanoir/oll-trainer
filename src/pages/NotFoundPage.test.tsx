/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { NotFoundPage } from './NotFoundPage';

describe('NotFoundPage', () => {
  it('snapshot', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <NotFoundPage />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
