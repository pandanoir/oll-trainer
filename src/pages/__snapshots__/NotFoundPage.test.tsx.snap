// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<DocumentFragment>
  <div
    class="_StyledDiv-sc-50fm8y-0 etxoIr"
  >
    <div
      class="_StyledDiv2-sc-50fm8y-1 byCwBT"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-face-sad-tear "
        data-icon="face-sad-tear"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 0C397.4 0 512 114.6 512 256C512 397.4 397.4 512 256 512C114.6 512 0 397.4 0 256C0 114.6 114.6 0 256 0zM256 352C290.9 352 323.2 367.8 348.3 394.9C354.3 401.4 364.4 401.7 370.9 395.7C377.4 389.7 377.7 379.6 371.7 373.1C341.6 340.5 301 320 256 320C247.2 320 240 327.2 240 336C240 344.8 247.2 352 256 352H256zM208 369C208 349 179.6 308.6 166.4 291.3C163.2 286.9 156.8 286.9 153.6 291.3C140.6 308.6 112 349 112 369C112 395 133.5 416 160 416C186.5 416 208 395 208 369H208zM303.6 208C303.6 225.7 317.1 240 335.6 240C353.3 240 367.6 225.7 367.6 208C367.6 190.3 353.3 176 335.6 176C317.1 176 303.6 190.3 303.6 208zM207.6 208C207.6 190.3 193.3 176 175.6 176C157.1 176 143.6 190.3 143.6 208C143.6 225.7 157.1 240 175.6 240C193.3 240 207.6 225.7 207.6 208z"
          fill="currentColor"
        />
      </svg>
       Not found
    </div>
    <div>
      すみません、お探しのページが見つかりませんでした…
    </div>
    <div>
      <a
        aria-current="page"
        class="PrimaryLinkButton-sc-z7yax6-0 ptYTb active"
        href="/"
      >
        Home に戻る
      </a>
    </div>
  </div>
</DocumentFragment>
`;
